pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -63,7 +63,7 @@ type CephClusterHealthCheckSpec struct {
 	// LivenessProbe allows changing the livenessProbe configuration for a given daemon
 	// +optional
 	LivenessProbe map[KeyType]*ProbeSpec `json:"livenessProbe,omitempty"`
-	// StartupProbe allows changing the startupProbe configuration for a given daemon
```

Comment:
Was this hunk added to this PR by accident? It does not seem to be related to the topic. Anyway, I think the singular"a given daemon" is perfectly fine here and does not need to be changed to plural. please revert.